tests/benchmark/run_benchmark_on_chrome.html

```diff
@@ -0,0 +1,453 @@
+            cell.textContent = run_time.toFixed(2);
+            return run_time;
+        }
+        function nbody_interface_time() {
+            const start_time = performance.now();
+            nbody_interface();
+            const end_time = performance.now();
+            const run_time = end_time - start_time;
+            const cell = document.getElementById(`nbody_interface_2`);
+            cell.textContent = run_time.toFixed(2);
+            return run_time;
+        }
+        function quicksort_time() {
+            const start_time = performance.now();
+            quicksort();
+            const end_time = performance.now();
+            const run_time = end_time - start_time;
+            const cell = document.getElementById(`quicksort_2`);
+            cell.textContent = run_time.toFixed(2);
+            return run_time;
+        }
+        function quicksort_float_time() {
+            const start_time = performance.now();
+            quicksort_float();
+            const end_time = performance.now();
+            const run_time = end_time - start_time;
+            const cell = document.getElementById(`quicksort_float_2`);
+            cell.textContent = run_time.toFixed(2);
+            return run_time;
+        }
+        function spectral_norm_time() {
+            const start_time = performance.now();
+            spectral_norm();
+            const end_time = performance.now();
+            const run_time = end_time - start_time;
+            const cell = document.getElementById(`spectral_norm_2`);
+            cell.textContent = run_time.toFixed(2);
+            return run_time;
+        }
+        function spectral_norm_i32_time() {
+            const start_time = performance.now();
+            spectral_norm_i32();
+            const end_time = performance.now();
+            const run_time = end_time - start_time;
+            const cell = document.getElementById(`spectral_norm_i32_2`);
+            cell.textContent = run_time.toFixed(2);
+            return run_time;
+        }
+
+        /* update js run times */
+        any_basic_type_access_time();
+        // any_complex_type_access_time();
+        array_access_time();
+        array_access_i32_time();
+        binarytrees_class_time();
+        // binarytrees_interface_time();
+        class_access_time();
+        class_allocation_time();
+        fibonacci_time();
+        interface_access_field_fastpath_time();
+        // interface_access_field_slowpath_time();
+        interface_access_method_fastpath_time();
+        // interface_access_method_slowpath_time();
+        mandelbrot_time();
+        mandelbrot_i32_time();
+        // merkletrees_time();
+        nbody_class_time();
+        nbody_interface_time();
+        quicksort_time();
+        quicksort_float_time();
+        spectral_norm_time();
+        spectral_norm_i32_time();
+
+        /* update wasm run times */
+        run_wasm_file('any_basic_type_access', warmupTimes);
```

Comment:
```suggestion
        await run_wasm_file('any_basic_type_access', warmupTimes);
```

Same for below lines